app/src/main/java/edu/metrostate/cardealer/activity/EditSpecificVehicle.java

```diff
@@ -0,0 +1,69 @@
+package edu.metrostate.cardealer.activity;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.annotation.SuppressLint;
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import edu.metrostate.cardealer.CarDealerApplication;
+import edu.metrostate.cardealer.R;
+import edu.metrostate.cardealer.inventory.Vehicle;
+import edu.metrostate.cardealer.adapter.VehicleAdapter;
+
```

Comment:
are you using both this activity and the vehicleListActivity to display a list of vehicles?

Reply:
Also please change the name of this class to include Activity in the name

Reply:
I tried to use the template that the professor had for the vehicle list, but I don't think that the vehicleListActivity is being used along with the EditSpecificVehicleActivity. (although I could be wrong)